pkg/controllers/management/node/controller.go

```diff
@@ -499,6 +501,28 @@ outer:
 	return obj, err
 }
 
+func (m *Lifecycle) checkScaledown(obj *v3.Node) error {
+	if obj.Spec.ScaledownTime == "" {
+		return nil
+	}
+
+	scaledown, err := time.Parse(time.RFC3339, obj.Spec.ScaledownTime)
+	if err != nil {
+		return fmt.Errorf("[node] failed to parse scaledown time, is it in RFC3339? %s", obj.Spec.ScaledownTime)
+	}
+
+	if scaledown.Before(time.Now()) {
```

Comment:
wouldn't we need to `EnqueueAfter` if the time is set to sometime later?  (we could maybe rely on the node updates which happen often, but wanted to ask if that's the decision)